configure.ac

```diff
@@ -4900,7 +4899,6 @@ echo "
 	BigArrays:     $enable_big_arrays
 	DTrace:        $enable_dtrace
 	LLVM Back End: $enable_llvm (dynamically loaded: $enable_loadedllvm)
-	Interpreter:   $enable_interpreter
```

Comment:
We should keep this

Reply:
if disabled, it would show in the `minimal` line later in the output. note that this is about the `configure` output, not `mono --version`; I will add something to the `mono --version` output.

Reply:
I know this is about configure output and we should show it there as well

Reply:
`./autogen.sh ...` would give you this:
```
        mcs source:    mcs
        C# Compiler:   roslyn

   Engine:
        Host:          x86_64-apple-darwin16.7.0
        Target:        x86_64-apple-darwin16.7.0
        GC:            sgen (concurrent by default) and Included Boehm GC with typed GC
        TLS:           pthread
        SIGALTSTACK:   yes
        Engine:        Building and using the JIT
        BigArrays:     no
        DTrace:        yes
        LLVM Back End: no (dynamically loaded: no)

   Libraries:
        .NET 4.x:        yes
        Xamarin.Android: no
        Xamarin.iOS:     no
        Xamarin.WatchOS: no
        Xamarin.TVOS:    no
        Xamarin.Mac:     no
        Windows AOT:     no
        Orbis:           no
        Unreal:          no
        WebAssembly:     no
        Test profiles:   AOT Full (no), AOT Hybrid (no)
        JNI support:     IKVM Native
        libgdiplus:      /Library/Frameworks/Mono.framework/Versions/Current/lib/libgdiplus.dylib
        zlib:            system zlib
        BTLS:            yes (x86_64)
```

`./autogen.sh --enable-minimal=interpreter ...`:
```
        mcs source:    mcs
        C# Compiler:   roslyn

   Engine:
        Host:          x86_64-apple-darwin16.7.0
        Target:        x86_64-apple-darwin16.7.0
        GC:            sgen (concurrent by default) and Included Boehm GC with typed GC
        TLS:           pthread
        SIGALTSTACK:   yes
        Engine:        Building and using the JIT
        BigArrays:     no
        DTrace:        yes
        LLVM Back End: no (dynamically loaded: no)

   Libraries:
        .NET 4.x:        yes
        Xamarin.Android: no
        Xamarin.iOS:     no
        Xamarin.WatchOS: no
        Xamarin.TVOS:    no
        Xamarin.Mac:     no
        Windows AOT:     no
        Orbis:           no
        Unreal:          no
        WebAssembly:     no
        Test profiles:   AOT Full (no), AOT Hybrid (no)
        JNI support:     IKVM Native
        libgdiplus:      /Library/Frameworks/Mono.framework/Versions/Current/lib/libgdiplus.dylib
        zlib:            system zlib
        BTLS:            yes (x86_64)
        Disabled:      interpreter

```

So I think that additional line is not really necessary?